yt_dlp/extractor/mediaset.py

```diff
@@ -310,23 +342,23 @@ class MediasetShowIE(MediasetIE):
             'id': '000000002763',
             'title': 'Le Iene',
         },
-        'playlist_count': 7,
+        'playlist_count_min': 7,
```

Comment:
It's `playlist_mincount` (as I remembered when a test unexpectedly succeeded with `playlist_min_count`).

Reply:
that's odd, why the test doesn't fail?  I'll fix it anyway.

Reply:
I think the test harness only looks for test conditions that it knows, since other keys at the same level of the test case are handled elsewhere: `test/test_download.py`